qt/python/mantidqt/widgets/workspacedisplay/status_bar_view.py

```diff
@@ -0,0 +1,45 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+from qtpy.QtCore import Qt, Signal, Slot
+from qtpy.QtWidgets import QMainWindow, QStatusBar
+
+from mantidqt.widgets.workspacedisplay.observing_view import ObservingView
+
+
+class StatusBarView(QMainWindow, ObservingView):
+    close_signal = Signal()
+    rename_signal = Signal(str)
+
+    def __init__(self, parent, central_widget, name, window_width=600, window_height=400, presenter=None):
```

Comment:
`window_size` as a `tuple` rather than separate parameters?

Reply:
I prefer separate parameters, I'm not sure we gain much from using a tuple, because then it's not immediately clear which is used as height and width